atm.js

```diff
@@ -1,21 +1,107 @@
+/*
 var checkingBalance = document.querySelector("div#checking_balance");
 checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+*/
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+window.onload = function(){
+  var checking = new Account(0);
+  var savings = new Account(0);
 
-}
 
-function checkingWithdrawal(amount){
+  var checkingDepositBtn = $("input[value='Deposit']").eq(0);
+  var checkingWithdrawBtn = $("input[value='Withdraw']").eq(0);
+  var checkingDisplay = $(".balance").eq(0);
+  var checkingInput = $("input[type='text']").eq(0);
 
```

Comment:
Since there was not any constraints against modifying the `html`, would recommend adding id's to assist with selection, in favor of using `eq` to improve readability